package.json

```diff
@@ -46,6 +46,9 @@
   "scripts": {
     "build": "tsc && copyfiles \"lib/**/*.json\" dist && copyfiles \"tests/**/*.js*\" dist && copyfiles \"package.json\" dist",
     "test": "npm run build && nyc jasmine --config=./tests/jasmine.json",
+    "publish:unstable": "./scripts/publish-unstable.sh",
+    "publish:release": "./scripts/publish.sh",
+    "version:release": "yarn version --minor --message \"release commit [skip ci]\"",
```

Comment:
Not a fan of the use of both yarn and npm in the same project.

Reply:
Ah nice catch sorry carried over from another project where we use yarn, def should not have both being used here. TBH I would rather see this project converted to using yarn